src/components/Party/PartyListItem.tsx

```diff
@@ -64,6 +65,7 @@ function PartyListItem({ party, expandedPartyId, setExpandedPartyId }: TPartyLis
 						</ChipContainer>
 						<PartyListItemTitleWrapper>
 							<Typography variant='h6'>{party.title}</Typography>
+							{party.isPrivate ? <LockIcon /> : null}
```

Comment:
```suggestion
							{party.isPrivate ? <LockIcon /> : <></>}
```
null보다 이게 더 좋을거 같은데요~

Reply:
제가 이곳저곳 null과 <></>를 섞어쓴 것 같습니다.
저부분에 null이 들어갈 경우 문제가 생길 여지가 있는건가요??

Reply:
문제는 안되는데 어색해서...